### ISSUE TITLE
👋 

### ISSUE THREAD

**Issue by samdeane** [Greatness7/tes3conv#1]
Not an issue per-se, just more of a hello. 

I've been working on something similar [elegantchaos/SkyrimFileFormat](https://github.com/elegantchaos/SkyrimFileFormat).

Is this project still active?

**Comment by samdeane** [comment]
It would be interesting to talk about the various design decisions... :)

**Comment by poperigby** [comment]
@samdeane there's also [DeltaPlugin](https://gitlab.com/bmwinger/delta-plugin) to look at. That's more actively maintained.

**Comment by samdeane** [comment]
Ooh interesting, thanks for the link. I stalled eventually on my stuff when real life intervened.

I think probably [Mutagen](https://github.com/Mutagen-Modding/Mutagen) and its associated tools are the way I will go now if I ever get back to it.

I do still think there's some interesting conversations to be had in terms of how to structure the files and tools, how to drive everything from a single set of definitions (eg shared with the TESEdit tools), and so on.

**Comment by poperigby** [comment]
Yeah Mutagen would be cool! Don't think it currently supports Morrowind though.
